<?xml version="1.0"?>

<!--|
    Copyright (C) 2019 Learning Algorithms and Systems Laboratory, EPFL, Switzerland
    Authors:  Harshit Khurana, Daan Stokbroekx
    email:    harshit.khurana@epfl.ch
    website:  lasa.epfl.ch

    This file is part of i_am_project EPFL (impact aware manipulation).
|-->

<launch>
  <arg name="robot_name_1" default="iiwa1"/>

  <!--This spawns iiwa7 and iiwa14 and starts both passive tracking in their namespace-->
  <include
      file = "$(find iiwa_toolkit)/launch/passive_track_gazebo.launch">    
      <arg name="robot_name_1" value="$(arg robot_name_1)"/>
      <arg name="iiwa_model_1" value="7"/>
      <!-- <arg name="position_1" value="-x 0.0 -y 0.0"/> -->
  </include>

  <!--Run AH_master to control position and velocity for game of air hockey-->
  <rosparam file="$(find i_am_project)/config/1v1_params.yaml" />
  <node name="initialization" pkg="i_am_project" type="AH_init" output="screen"/>
  <node name="trail_initialization" pkg="i_am_project" type="AH_trail_init" output="screen"/>
  <node name="master" pkg="i_am_project" type="AH_master"   output="screen"/>

</launch>

